build/Targets/Settings.props

```diff
@@ -158,7 +158,7 @@
   <!-- Unix specific settings -->
   <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
     <RoslynDebugType>portable</RoslynDebugType>
-    <RoslynPortableRuntimeIdentifiers>ubuntu.14.04-x64;ubuntu.16.04-x64;osx.10.12-x64</RoslynPortableRuntimeIdentifiers>
+    <RoslynPortableRuntimeIdentifiers>linux-x64;osx-x64</RoslynPortableRuntimeIdentifiers>
```

Comment:
Do we need to keep this is as -x64 still? Don't want to over complicate this change but curiosu if we should explore removing that too.

Reply:
No idea... I'll run a local build and see if it explodes, and see how bad it is 🙂